source/common/network/proxy_protocol_filter_state.cc

```diff
@@ -9,5 +12,66 @@ const std::string& ProxyProtocolFilterState::key() {
   CONSTRUCT_ON_FIRST_USE(std::string, "envoy.network.proxy_protocol_options");
 }
 
+class ProxyProtocolFilterStateReflection : public StreamInfo::FilterState::ObjectReflection {
+public:
+  ProxyProtocolFilterStateReflection(const ProxyProtocolFilterState* object) : object_(object) {}
+
+  FieldType getField(absl::string_view tlv_type_str) const override {    
```

Comment:
[q] I see that getField() is meant to access all field-level data, not just TLVs. However, TLVs are the only required sub-field for our use case. Given the complexity of this filter state compared to others, is this design acceptable? It could be extended in the future to support formatting other fields, like local / remote IP / port. (I can imagine a future extension supporting something like getField("TLV:5") and enabling access to all other fields.)